tests/adapters/switches/juniper_test.py

```diff
@@ -4312,11 +4312,11 @@ def test_get_bond(self):
 
         assert_that(if3.number, equal_to(3))
         assert_that(if3.link_speed, equal_to('1g'))
-        assert_that(if3.interface.name, equal_to("ae3"))
-        assert_that(if3.interface.port_mode, equal_to(TRUNK))
-        assert_that(if3.interface.access_vlan, equal_to(None))
-        assert_that(if3.interface.trunk_native_vlan, equal_to(2000))
-        assert_that(if3.interface.trunk_vlans, equal_to([999, 1000, 1001]))
+        assert_that(if3.name, equal_to('ae3'))
```

Comment:
Si after the deprecation period of the bond.interface.name, bond.name will be None?   caus if it's not then cached switch can't cache a new bond and we'll have done this whole thing for nothing